cli/src/config/templates.toml

```diff
@@ -17,9 +17,8 @@ commit_summary = 'format_commit_summary_with_refs(self, bookmarks)'
 annotate_commit_summary = '''
 separate(" ",
   format_short_id(change_id),
-  format_short_id(commit_id),
-  format_short_signature(author),
   format_timestamp(committer.timestamp()),
```

Comment:
and to people who use `.shortest(0)`. I assume they'll probably want to customize the annotation template at all.

Reply:
Yes. Maybe, until padding exists, we should just hard-code everything here and leave a note to use the normal macros once the padding is available?

(This is still optional)

Reply:
>  we should just hard-code everything here

@allonsy, was that the reason you didn't use `format_*()` aliases in the original patch?

I don't have a strong opinion, but I thought it would be nice to provide some customization points.

Reply:
I think you give me too much credit :)
I didn't include the format_* because I wasn't as aware of them as I am now when I started work on file annotation